proto/dolt/services/remotesapi/v1alpha1/chunkstore.proto

```diff
@@ -169,4 +171,14 @@ message TableFileInfo {
 message ListTableFilesResponse {
   bytes root_hash = 1;
   repeated TableFileInfo table_file_info = 2;
+}
+
+message AddTableFilesRequest {
+  RepoId repo_id = 1;
+  ClientRepoFormat client_repo_format = 2;
+  repeated ChunkTableInfo chunk_table_info = 3;
+}
+
+message AddTableFilesResponse {
```

Comment:
I think it's more customary to return Empty or throw an exception, rather than make a client check a success flag.

Reply:
yes.  will fix this and commit at a later time.